apps/api/v2/src/modules/teams/memberships/inputs/create-team-membership.input.ts

```diff
@@ -1,20 +1,23 @@
-import { ApiProperty } from "@nestjs/swagger";
+import { ApiProperty, ApiPropertyOptional } from "@nestjs/swagger";
 import { MembershipRole } from "@prisma/client";
 import { IsBoolean, IsOptional, IsEnum, IsInt } from "class-validator";
 
 export class CreateTeamMembershipInput {
   @IsInt()
+  @ApiProperty({ type: Number })
   readonly userId!: number;
 
   @IsOptional()
   @IsBoolean()
+  @ApiPropertyOptional({ type: Boolean, default: false })
   readonly accepted?: boolean = false;
 
   @IsEnum(MembershipRole)
-  @ApiProperty({ enum: ["MEMBER", "OWNER", "ADMIN"] })
+  @ApiProperty({ enum: ["MEMBER", "OWNER", "ADMIN"], default: "MEMBER" })
```

Comment:
Here we need api property optional and IsOptional because we are setting default value on line 17:

```
readonly role: MembershipRole = MembershipRole.MEMBER;
```